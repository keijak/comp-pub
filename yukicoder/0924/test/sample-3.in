20 10
-574 627 212 956 -285 161 -331 -107 -229 -87 -414 -917 842 835 566 117 -298 -608 -964 -182
2 10
9 10
11 12
4 12
19 19
6 17
14 15
1 13
5 13
16 20
