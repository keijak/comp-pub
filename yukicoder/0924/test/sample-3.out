2908
142
503
2870
0
4730
269
5461
2756
1507
